CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to the Pony compiler and standard library will be documented
 
 ### Changed
 
+- Remove `glob` package from standard library ([PR #3219](https://github.com/ponylang/ponyc/pull/3219))
```

Comment:
The PR link is incorrect, is there a reason not to just label the PR?

Reply:
Yes. The bot is not working.

Reply:
Ah, ok. I wasn't aware of that

Reply:
Updated.